gradoop-flink/src/main/java/org/gradoop/flink/model/api/operators/LogicalGraphOperators.java

```diff
@@ -344,6 +346,115 @@ LogicalGraph reduceOnNeighbors(
    */
   DataSet<Boolean> equalsByData(LogicalGraph other);
 
+  /**
+   * Returns a graph with the same structure but a specified property of an element is rolled up
+   * by the declared function. It is possible to roll up either on one vertex / edge type or all
+   * vertex / edge types. Additionally the rolled up value can be stored under a new key. If the
+   * original key shall be reused the old value is stored under the key 'key__x' where 'x' is a
+   * version number. This number increases on every continuous roll up call where the highest
+   * number is the level direct below the rolled up one.
+   *
+   * @param rollUp roll up operation
+   * @return logical graph with rolled up properties
+   */
+  LogicalGraph rollUp(RollUp rollUp);
```

Comment:
remove this method (can be done by `LogicalGraph.callForGraph(DrillBuilder...buildVertex())`